README.md

```diff
@@ -12,7 +12,7 @@ information but not when your trigger event is `push`. This action enables to ge
 ```
   steps:
     - uses: actions/checkout@v1
-    - uses: 8BitJonny/gh-get-current-pr@v1
+    - uses: 8BitJonny/gh-get-current-pr@v1.0.1
```

Comment:
You still have a `v` there, but the tag name doesn't have a `v`.
it should just be `gh-get-current-pr@1.0.1`

Reply:
Man I really shouldn't be committing so late late at night. Thx man

Reply:
Fixed in https://github.com/8BitJonny/gh-get-current-pr/commit/bcb9bb2bfcb3f90c4536e8df317f9f820aeeb276